test/Conversion/amd/async-ops-alias-scopes.mlir

```diff
@@ -124,3 +124,38 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.targ
     tt.return
   }
 }
+
+// -----
+
+// COMMON: [[LOCAL_LOAD_SCOPE:#.*]] = #llvm.alias_scope<id = "amdgpu.LocalLoads"
+// COMMON: [[ASYNC_COPY_SCOPE:#.*]] = #llvm.alias_scope<id = "amdgpu.AsyncCopies"
+#blocked = #ttg.blocked<{sizePerThread = [1, 1], threadsPerWarp = [64, 1], warpsPerCTA = [1, 1], order = [0, 1]}>
+#shared = #ttg.swizzled_shared<{vec = 8, perPhase = 8, maxPhase = 2, order = [0, 1]}>
+#shared1 = #ttg.swizzled_shared<{vec = 8, perPhase = 1, maxPhase = 16, order = [1, 0]}>
+#smem = #ttg.shared_memory
+#mma = #ttg.amd_mfma<{versionMajor = 4, versionMinor = 0, warpsPerCTA = [1, 4], instrShape = [32, 32], isTransposed = true}>
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 1 : i32, ttg.target = "hip:gfx950", "ttg.threads-per-warp" = 64 : i32} {
+  tt.func public @local_loads_with_loop_carried_token(%arg0: !tt.ptr<f16> {tt.divisibility = 16 : i32, tt.pointer_range = 32 : i32},
```

Comment:
Can you add `// COMMON-LABEL` to separate sections?

Reply:
We match the ids of the alias scopes before the function starts. So if I add `COMMON-LABEL` the variables `LOCAL_LOAD_SCOPE` and `ASYNC_COPY_SCOPE` will be undefined when matching inside the function.

That's why I added `// COMMON: llvm.return` at the end of each test as a guard

Reply:
You can do `[[$...]]` to make the capture a global variable. See https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-string-substitution-blocks.

Reply:
It's a minor issue. Given you are offline already I just went ahead and added 36031ba for it. :)